files/root/grafana_tools/backup_and_sync/sync_grafana.sh

```diff
@@ -6,7 +6,7 @@ pushd $(dirname "$0") > /dev/null
 source ./config
 
 ## Get Grafana backing DB type
-db_type=$(grep "^type = " /etc/grafana/grafana.ini | cut -d' ' -f 3)
+db_type=$(grep "^type = " /etc/grafana/grafana.ini | cut -d' '- f 3)
```

Comment:
looks like somehow a typo got introduced

Reply:
fixed in latest commit